mojo/stdlib/stdlib/builtin/simd.mojo

```diff
@@ -2482,13 +2482,41 @@ struct SIMD[dtype: DType, size: Int](
     # Reduce operations
     # ===------------------------------------------------------------------=== #
 
+    alias _T = SIMD[dtype, _]
+
+    # TODO: remove when non-capturing can be converted to capturing.
```

Comment:
Can you please file a GitHub issue for this so the compiler team and @dgurchenkov know about this request? Thanks!